tests/functional/c/comparison_of_constants.py

```diff
@@ -0,0 +1,51 @@
+# pylint: disable=missing-docstring, comparison-with-itself
+
+
+if 2 is 2: # [literal-comparison, comparison-of-constants]
```

Comment:
Raising two messages for the same piece of code might be an issue. I don't have a good solution to propose though.

Reply:
Why might this be an issue?

Reply:
Well it's two messages instead of one so it could be considered noise/pylint being inconsistent. But the votes on https://github.com/PyCQA/pylint/pull/6413#pullrequestreview-961130709 say it isn't an issue :)